Dockerfile

```
@@ -57,6 +57,7 @@ RUN directory-artifact-gen \
     -t raspberrypi3_64 \
     -t raspberrypi4 \
     -t raspberrypi4_64 \
+    -t raspberrypi5_64 \
```

The UI will likely use `raspberrypi5` (no 64 prefix) as it does today for Raspberry Pi 3 and 4. Let's add both to be safe 🙏 
```suggestion
    -t raspberrypi5 \
    -t raspberrypi5_64 \
```